automatminer/automl/tests/test_nn.py

```diff
@@ -0,0 +1,65 @@
+import os
+import unittest
+
+import pandas as pd
+from sklearn.metrics import r2_score, f1_score
+
+from automatminer.automl.adaptors import NeuralNetworkAdaptor
+from automatminer.utils.package_tools import AutomatminerError
+
+__author__ = ['Qi Wang <qwang3@lbl.gov>', 'Alex Dunn <ardunn@lbl.gov>']
+
+
+@unittest.skipIf("CI" in os.environ.keys(), "Test too intensive for CircleCI.")
```

Comment:
Are these tests actually too intensive for CircleCI? If so it's fine to skip for now. The TPOT ones are excluded because tpot isn't that great with handling memory or the model search space, even when using debugging settings.

Reply:
also, add yourself to the authors again!